### DISCUSSION TITLE
[Feature] Album-Based Search

### DISCUSSION THREAD

**Discussion by claytongearhart** [immich-app/immich#10032] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, the search functionality on the main `/photos` page does not support querying based on the presence or absence of photos in specific albums. It'd be nice to enable complex queries using boolean logic, such as `(in:albumA AND in:albumB) AND NOT in:albumC`. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2407
